CHANGELOG.md

```diff
@@ -12,6 +12,10 @@
 * Adjusted gallery tests to not fail on deprecation warnings from pandas. @rly [#157](https://github.com/hdmf-dev/hdmf-zarr/pull/157)
 * Fixed bug in `pyproject.toml` where duplicate versions of `numcodec` where specified. @oruebel [#163](https://github.com/hdmf-dev/hdmf-zarr/pull/163)
 
+### Internal Fixes
+* Updated actions in `deploy_release.yml` workflow to avoid use of deprecated Node.js 16 . @oruebel [#165](https://github.com/hdmf-dev/hdmf-zarr/pull/165)
```

Comment:
```suggestion
* Updated actions in `deploy_release.yml` workflow to avoid use of deprecated Node.js 16. @oruebel [#165](https://github.com/hdmf-dev/hdmf-zarr/pull/165)
```